client/client_test.go

```diff
@@ -56,7 +56,7 @@ func TestClientIntegration(t *testing.T) {
 func testRelativeWorkDir(t *testing.T, sb integration.Sandbox) {
 	t.Parallel()
 	requiresLinux(t)
-	c, err := New(sb.Address())
+	c, err := New(context.TODO(), sb.Address())
```

Comment:
maybe better to define `func testContext() context.Context`?

Reply:
`context.TODO()` is already widely used in this test code.